release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -84,6 +84,13 @@ The {product-title} 4.10 installer uses new default component types for installa
 * AWS EC2 M6i instances for both control plane and compute nodes, where available
 * AWS EBS gp3 storage
 
+[id="ocp-4-10-enhancements-to-API-in-installconfig"]
+==== Enhancements to API in install.config
+
+When a user installs OCP on a Baremetal IPI platform, they have nowhere to configure custom network interfaces, such as static IPs or a vLAN to communicate with the Ironic server.
```

Comment:
I feel like the wording on this line is a little misleading.
maybe add "formerly..." or something to clarify this was the case in prior versions.

Reply:
Updated as suggested.